### ISSUE TITLE
Totally a feature request...IGNORE function

### ISSUE THREAD

**Issue by jbogie-pivotal** [genevieve/leftovers#68]
Hey Y'all!

Would love to see some sort of --ignore option built in!

Example: Use an input file of resources that leftovers would ignore when running...
```
leftovers --no-confirm --ignore ~/ignored_resources.txt

cat ignored_resources.txt

"IAM Service Account: projects/PROJECT/serviceAccounts/pks-v1-tform@PROJECT.iam.gserviceaccount.com"
"IAM Service Account: projects/PROJECT/serviceAccounts/410024626587-compute@developer.gserviceaccount.com"
```
Now I can keep some resources around on each of the cloud providers that I don't want deleted but totally nuke the rest of the environment.

**Comment by tammersaleh** [comment]
I was looking for something similar but was thinking more along the lines of a tag I could put on resources that I didn't want to be removed.

**Comment by genevieve** [comment]
Hey @jbogie-pivotal & @tammersaleh!

Given the current implementation of leftovers, the easiest/quickest solution would be to allow `leftovers --ignore TYPE` where `TYPE` could be anything from the `leftovers types` list. Would this cover most of your use-cases or would it only be useful if it was on a specific-resource-name basis?

What each IaaS leftovers bubbles up to the top is a `name` that has been generated to include pertinent information (for instance, some resources don't have clear names so I append details like tags or network name to help with filtering them.) This means that it would be more difficult to do an "exact match" between resources in a `resources.txt` and the names in the list.

**Comment by jbogie-pivotal** [comment]
@genevieve doing it by `type` still wouldn't get around the the challenge I'm facing. @tammersaleh has a great idea. Is it possible to ignore by `tag`?

I could absolutely change my process to create custom tags for 'special' resources and then anytime I cleanup IaaS just use the `--ignore tag-foo` functionality to spare the stuff I want to keep around.

**Comment by tammersaleh** [comment]
Yeah, I'm not sure limiting by resource type would be helpful for my use case either.
